src/main/java/easton/model/Event.java

```diff
@@ -0,0 +1,48 @@
+package easton.model;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import easton.exception.DateTimeFormatException;
+
+/**
+ * Represents an event task.
+ */
+public class Event extends Task {
+
+    private LocalDateTime from;
+    private LocalDateTime to;
```

Comment:
While variable name `to` is functional, I think you can consider renaming it to something more descriptive for better clarity, like `endDate` or `toDate`.